test/README.md

```diff
@@ -95,6 +95,9 @@ You can set these environment variables to configure the test suite:
                      "firefox"
                   "chromium" is the default.
 
+    TEST_SHOW_BROWSER  Set to `True` to run browser visibly. When not specified,
```

Comment:
`True` is a weird value for shell boolean variables. Usually one just tests non-emptiness.

"visibly" → "interactively" perhaps?